scripts/release/update_stable_ids/update_stable_ids.pl

```diff
@@ -45,13 +46,17 @@
 # Make sure our requested DBs are slice DBs
 check_db_names();
 
-back_up_databases(
-    [$user, $pass, $gk_central, $ghost],
-    [$user, $pass, $release_db, 'localhost']
-);
+print "host is $host\n";
+print "g_host is $ghost\n";
+
+#back_up_databases(
+#    [$user, $pass, $gk_central, $ghost],
+#    [$user, $pass, $release_db, $host]
+#);
 
 get_api_connections()->{$release_db}->execute("START TRANSACTION");
 get_api_connections()->{$gk_central}->execute("START TRANSACTION");
+print "transactions started\n";
```

Comment:
Does this need to be logged?

Reply:
LOL probably debugging output. I'll remove that.